docs: fix some typos (#849)
bareos/bareos: docs/manuals/source/DeveloperGuide/api.rst

    .api 1

or (from times where they was only one API flavour)

::

    .api

> This mode is intended to create output that is earlier to parse.

Reviewer: ```suggestion
This mode is intended to create output that is easier to parse.
```

> Internaly some commands vary their output for API mode 1, but not
> all.

Reviewer: ```suggestion
Internaly some commands vary their output for API mode 1.
```

> In API mode 1 some output is only delimited by the end of a packet, not
> by a new line. bconsole does not display the end of packets (for good
> reason, as some output (e.g. :bcommand:`status`) is sent in multiple packets).
> If running in a bconsole, this leads to unparsable output for human.

Reviewer: ```suggestion
In API mode 1 some output is delimited by end of a packet and not by a newline character.
Thus the output of API mode 1 may be non-parseable when being used in bconsole, as end of packet is not displayed there.
```

Author: If I remember correctly, the output is displayed. It is just very hard to read, as line-breaks are missing.

   make it easier to implement a full JSON-RPC service later.

> -  No user interaction inside a command (meaning: if not all parameter

Reviewer: ```suggestion
-  No user interaction inside a command (meaning: if not all parameters
```

   are given to a :bcommand:`run` command, the command fails).

> -  Each command creates exactly one responce object.

Reviewer: ```suggestion
-  Each command creates exactly one response object.
```

Currently a subset of the available commands return their result in JSON

> format, while others still write plain text output. When finished, it
> should be safe to run all commands in JSON mode.

Reviewer: ```suggestion
format, while others still write plain text output.
```

Introduction
~~~~~~~~~~~~

> This is an experimental and preliminary backend for development purposes. The endpoints are subject to review and may be changed in the future.

Reviewer: ```suggestion
This is an experimental and preliminary backend for development purposes. The endpoints are subject to review and may change in the future.
```
